CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+# Contributing
+
+This document provides guidelines for contributing to this repository.
+
+## Development Environment Setup
+
+### Prerequisites
+* Basic tools: `curl`, `jq`, `oc`
+* go: v1.19
+* Container engine: `podman`
+
+### Setting up the Environment
+For initial setup, please refer to the
+[Running a Test Environment](./README.md#running-a-test-environment) section
```

Comment:
now that we have this document, that section belongs over here - and we should have a reference there

Reply:
Done

Reply:
@srivickynesh did you forget to push your changes? That section is still in the README.

Reply:
Done. Pushed the correct commit